src/DeviceInterface/InterfacePrototype.js

```diff
@@ -47,6 +47,9 @@ class InterfacePrototype {
     /** @type {PasswordGenerator} */
     passwordGenerator = new PasswordGenerator();
 
+    /** @type {import("../InContextSignup.js").InContextSignup | null} */
+    inContextSignup = null
```

Comment:
`null` because not all devices support this yet, and TS helps by forcing us to check for `null`